components/SlackAuthor.js

```diff
@@ -14,6 +14,10 @@ class SlackAuthor extends SlackComponent {
     this.props = props;
   }
 
+  addParentAttachment(keyName, value) {
```

Comment:
`add` feels that's incrementing. Let's rename it to `set` as it feels more "imperative"